src/everest/config/everest_config.py

```diff
@@ -753,14 +787,33 @@ def lint_config_dict_with_raise(config: dict):
         EverestConfig.model_validate(config)
 
     @staticmethod
-    def load_file(config_path: str) -> "EverestConfig":
-        config_path = os.path.realpath(config_path)
+    def load_file(config_file: str) -> "EverestConfig":
+        config_path = os.path.realpath(config_file)
 
         if not os.path.isfile(config_path):
             raise FileNotFoundError(f"File not found: {config_path}")
 
         config_dict = yaml_file_to_substituted_config_dict(config_path)
-        return EverestConfig.model_validate(config_dict)
+        try:
+            return EverestConfig.model_validate(config_dict)
+        except ValidationError as error:
+            exp = EverestValidationError()
+            file_content = []
+            with open(config_path, encoding="utf-8") as f:
+                file_content = f.readlines()
+
+            for e in error.errors(
+                include_context=True, include_input=True, include_url=False
+            ):
+                if e["type"] == "literal_error":
```

Comment:
Why is this? Do literal errors for some reason not include position information?